src/pages/Wishlist/Wishlist.js

```diff
@@ -1,267 +1,89 @@
-                  alt="Travel Bags"
-                  class="grid-col-img cards-img"
-                />
-              </div>
-
-              <div className="card-content padding-s">
-                <h3 className="card-grid">
-                  Stylish Bags
-                  <span className="cards-rating">
-                    4.5<i className="fa-solid fa-star star-rating"></i>
-                  </span>
-                </h3>
-
-                <div className="sm-gap">
-                  <span className="txt-bold"> Rs.1155</span>
-                  <span className="txt-crossed-off">Rs.1299</span>
-                  <span className="txt-high-light">10% Off</span>
-                </div>
-
-                <div class="card-footer">
-                  <Link to="/cart">
-                    <button class="btn btn-text-icon-primary grid-cards-icons">
-                      <span class="btn-card-icon">
-                        <i class="fas fa-shopping-cart"></i>{" "}
-                      </span>
-                      Move to Cart
-                    </button>
-                  </Link>
-                </div>
-              </div>
-              <span class="tag">New</span>
-              <button type="button" class="btn-dismissal">
-                <i class="fa-solid fa-xmark card-btn-dismissal"></i>
-              </button>
-            </div>
-          </li>
-
-          <li class="section-main">
-            <div class="cards">
-              <div class="img-container">
-                <img
-                  src="https://github.com/ranitasahaa1230/Trekkart-Store/blob/d51db72677ff63c4de681f8b249d14fcab32f3ac/components/images/luggage2.jpg?raw=true"
-                  alt="hiking"
-                  class="grid-col-img cards-img"
-                />
-              </div>
-
-              <div className="card-content padding-s">
-                <h3 className="card-grid">
-                  Stylish Bags
-                  <span className="cards-rating">
-                    5<i className="fa-solid fa-star star-rating"></i>
-                  </span>
-                </h3>
-
-                <div className="sm-gap">
-                  <span className="txt-bold"> Rs.399</span>
-                  <span className="txt-crossed-off">Rs.599</span>
-                  <span className="txt-high-light">10% Off</span>
-                </div>
 
-                <div class="card-footer">
-                  <Link to="/cart">
-                    <button class="btn btn-text-icon-primary grid-cards-icons">
-                      <span class="btn-card-icon">
-                        <i class="fas fa-shopping-cart"></i>{" "}
+        <ul className="wishlist-products">
+        {isWishlistHasItem ? (
+          wishList.map((product) => (
+              <li className="section-main" key={product._id}>
+                <div className="cards">
+                  <div className="img-container">
+                    <img
+                      src={product.img}
```

Comment:
over here as well. you can destructure product

Reply:
Will do it